substrate/frame/core-fellowship/src/weights.rs

```diff
@@ -79,7 +79,7 @@ pub trait WeightInfo {
 	fn set_active() -> Weight;
 	fn induct() -> Weight;
 	fn promote() -> Weight;
-	fn promote_fast(r: u32, ) -> Weight;
+	fn promote_fast(r: u16, ) -> Weight;
```

Comment:
@Doordashcon Changed this back to `u32` because `Linear` currently supports only `u32`. Using `u16` here caused a compilation issue when generating weights, more info and fix: https://github.com/paritytech/polkadot-sdk/pull/9516.